### PR TITLE
Patrick clean up

### PR DESCRIPTION
**Pull request by patrickmohrmann**

### CODE REVIEW

File: module/config.mjs
Comment (ChristopherSD) [inline module/config.mjs:1]: Any reason the whitespace is different here than the rest of file?

Comment (patrickmohrmann) [reply module/config.mjs:1]: what whitespace?

File: module/data/actor/pc.mjs
Comment (patrickmohrmann) [inline module/data/actor/pc.mjs:1]: this fixed #292

File: module/config.mjs
Comment (ChristopherSD) [reply module/config.mjs:1]: okay, as long as we're doing it consistently across the file, either one is fine for me

**Review by ChristopherSD — APPROVED** [review]

**Pull request merged**